src/utils/openCloseComponent.ts

```diff
@@ -100,13 +114,15 @@ export function onToggleOpenCloseComponent(component: OpenCloseComponent): void
 /**
  * Helper to keep track of transition listeners on setTransitionEl and connectedCallback on OpenCloseComponent components.
  *
+ * For component which do not have open prop, use `onToggleOpenCloseComponent` implementation.
+ *
  * @param component
  */
 export function connectOpenCloseComponent(component: OpenCloseComponent): void {
   disconnectOpenCloseComponent(component);
   if (component.transitionEl) {
     const boundOnTransitionStart: (event: TransitionEvent) => void = transitionStart.bind(component);
-    const boundOnTransitionEnd: (event: TransitionEvent) => void = transitionEnd.bind(component);
+    const boundOnTransitionEnd: (event: PageTransitionEventInit) => void = transitionEnd.bind(component);
```

Comment:
We're not going to be needing this anymore, since we're removing the connect logic altogether, along with the comment above.

Reply:
i think this was a typo.

Reply:
Sorry, I know it was a draft.